src/darwin/frida-helper-backend-glue.m

```diff
@@ -78,7 +78,7 @@
     goto bsd_failure; \
   }
 
-#ifdef HAVE_IOS
+#if defined(HAVE_IOS) || defined(HAVE_TVOS)
```

Comment:
```suggestion
#if defined (HAVE_IOS) || defined (HAVE_TVOS)
```

Here and elsewhere. Style guide can be found [here](https://github.com/frida/frida/blob/main/CONTRIBUTING.md).